app/views/surveys/_form.html.erb

```diff
@@ -68,7 +62,7 @@
         Stimmen abgegeben wurden.
       </p>
 
-      <p><small>(min. 2, max. 10, Zeichenlänge max. 150, in deutsch und polnisch)</small></p>
+      <p><small>(min. 2, max. 10, Zeichenlänge max. 150, in <%= @languages.join(" und ") %>)</small></p>
```

Comment:
bin auf einen Fehler gestoßen, den ich mir jetzt nicht genauer angucken kann

![Bildschirmfoto 2021-07-07 um 10 17 43](https://user-images.githubusercontent.com/1942953/124725352-14b66a00-df0d-11eb-992c-a87740059bb9.png)

Reply:
ich pack da eine prüfung auf @languages dazu.

Reply:
und was, wenn es hier mehr als 2 sprachen gäbe. dann steht da "sprache 1 und sprache 2 und sprache 3 und sprache 4"? das ist auch noch nicht ausgeklügelt.

Reply:
mehrsprachigkeit ist allgemein noch nicht ausgeklügelt.
wir haben an einigen stellen grundlagen dafür gelegt, aber an anderen mussten wir es hardcoden, damit wir nicht (ohne dass es beauftragt ist) unheimlich viel mehraufwand haben

Reply:
nach aktueller hinterlegung auf int/development würde hier auch "de und pl" rauskommen, statt "deutsch und polnisch".

Reply:
quickfix für den bug done in 614602e